src/openforms/js/components/admin/form_design/utils.js

```diff
@@ -57,6 +57,74 @@ const checkKeyChange = (mutationType, newComponent, oldComponent) => {
   return newComponent.key !== oldComponent.key;
 };
 
+const checkIfInitialValueTransformationIsNeeded = (newType, originalVariable) => {
```

Comment:
we only use the `initialValue` of the variable - do we need to pass the entire variable then? Looks like `originalValue` would be sufficient as an argument.

Reply:
Yeah, the entire object is indeed not needed.. I've changed it 👍